tests/unit/recommenders/evaluation/test_python_evaluation_time_performance.py

```diff
@@ -185,7 +185,7 @@ def test_python_ndcg_at_k(rating_true, rating_pred):
             col_prediction=DEFAULT_PREDICTION_COL,
             k=10,
         )
-    assert t.interval < 21.55627936 * (1 + TOL)
```

Comment:
The new version takes more time, because for raw and exp scores, we need to sort the dataframe based on the true ratings. We can probably use the old version for binary, and new version for raw and exp.